### PR TITLE
ensure ordered output in message printed by --list-prs, drop unused GITHUB_MAX_PER_PAGE from _postprocess_list_prs

### PR DESCRIPTION
**Pull request by boegel**
@migueldiascosta Some final minor changes for https://github.com/easybuilders/easybuild-framework/pull/2400 ...

### CODE REVIEW

File: test/framework/options.py [hunk 1]
```diff
@@ -3216,8 +3216,7 @@ def test_list_prs(self):
 
         args = ['--list-prs', 'closed,updated,asc']
         txt, _ = self._run_mock_eb(args, testing=False)
-        expected = "Listing PRs with parameters "
-        expected += "{'sort': 'updated', 'per_page': %s, 'state': 'closed', 'direction': 'asc'}" % GITHUB_MAX_PER_PAGE
+        expected = "Listing PRs with parameters: direction=asc, per_page=100, sort=updated, state=closed"
```
Comment (migueldiascosta) [inline test/framework/options.py:3219]: but the explicit `per_page=100` will fail if `GITHUB_MAX_PER_PAGE` is ever changed...

unless we allow specifying `per_page` as an option an set/test it explicitly?

Comment (boegel) [reply test/framework/options.py:3219]: GITHUB_MAX_PER_PAGE is a constant, it shouldn’t change (and we didn’t come up with the value, it’s a max set by GitHub, which is unlikely to change).

Hardcoding of values of tests is often done, exactly for the reason you mention: if something is changed in-place, the tests should catch it, not just follow along...

**Review by migueldiascosta — APPROVED** [review]
lgtm :)

**Comment by migueldiascosta** [comment]
going in, thanks @boegel

**Pull request merged**